modules/@apostrophecms/schema/ui/apos/components/AposInputRange.vue

```diff
@@ -0,0 +1,248 @@
+<template>
+  <AposInputWrapper
+    :modifiers="modifiers" :field="field"
+    :error="effectiveError" :uid="uid"
+  >
+    <template #body>
+      <div class="apos-input-wrapper">
+        <div class="apos-range-input">
+          <input
+            type="range"
+            :min="field.min"
+            :max="field.max"
+            :step="field.step"
+            class="apos-range"
+            v-model="next"
+            :id="uid"
+          >
+          <div class="apos-range-input__scale">
+            <span class="apos-range-input__scale-value apos-range-input__scale-value--min">
```

Comment:
You don't use the `--min` and `--max` classes. You could also get away with first-child and last-child if you needed to. Let's remove these classes.

Reply:
Also doesn't look like `apos-range-input__scale-value` are used. I guess the spans are needed for flexing. If they're there we should have some visually hidden labeling (Min: Max:)